vllm/model_executor/layers/fused_moe/rocm_aiter_fused_moe.py

```diff
@@ -89,7 +143,29 @@ def rocm_aiter_fused_experts(
         )
         return out_asm
 
+    elif is_rocm_aiter_channel_scaled_moe_enabled() and use_fp8_w8a8:
+
+        # # All AITER Fused MoE kernels are expecting the following datatypes
+        # topk_weights = topk_weights.to(torch.float32)
+        # topk_ids = topk_ids.to(torch.int32)
+
+        return rocm_aiter_asm_moe_tkw1(hidden_states,
```

Comment:
Let's assert `apply_router_weight_on_input=True` or do the if branch check when calling the `_tkw1` kernel? btw, we should have some comments to illustrate the difference between `_tkw1` kernel and other aiter kernels. The difference is on applying topk_weights on the output of the first GEMM or the second GEMM